### ISSUE TITLE
[Bug]: Picker with help-text is not announced by screen readers

### ISSUE THREAD

**Issue by Rajdeepc** [adobe/spectrum-web-components#5361]
### Code of conduct

- [x] I agree to follow this project's code of conduct.

### Impacted component(s)

Picker

### Expected behavior

https://opensource.adobe.com/spectrum-web-components/components/picker/#use-help-text-to-show-context
The help-text should be announced by screen-readers and be able to both render and get an accessible association for help text with `sp-picker`

### Actual behavior

https://opensource.adobe.com/spectrum-web-components/components/picker/ (help text section at bottom)
This doesn’t work as the help text isn’t in the same shadow so the describedby can’t resolve

https://opensource.adobe.com/spectrum-web-components/components/help-text/#ensure-help-text-and-field-share-the-same-root
Despite it saying that sp-picker supports a help-text slot the sp-picker [documentation](https://opensource.adobe.com/spectrum-web-components/components/picker/api/) doesn’t agree

### What browsers are you seeing the problem in?

Microsoft Edge, Safari, Chrome, Firefox

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/picker/#use-help-text-to-show-context
2. Open NVDA or JAWS screen readers
3. Check the help-text is not announced by screen reader

### Severity

SEV 4

**Comment by Rajdeepc** [comment]
SWC-790

**Comment by baliya367** [comment]
This fix in another issue and it might help you [click](https://newrides.es/captha/)
